src/logistration/data/service.js

```diff
@@ -50,9 +50,10 @@ export async function login(creds) {
 export async function getThirdPartyAuthContext(urlParams) {
   const requestConfig = {
     headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
+    isPublic: true,
   };
 
-  const { data } = await getHttpClient()
+  const { data } = await getAuthenticatedHttpClient()
```

Comment:
This is necessary to associate the subsequent request with backend session so that we can get the "current provider" in the pipeline and user details.
If I use `getHttpClient`, TPA context request was unable to get this information as it always considered it an anonymous new user

Reply:
Make sense, thank you for clarifying.